blog/2023-08-21-autonomous-localization.md

```diff
@@ -0,0 +1,109 @@
+
+## So, how can I get my strings translated?
+To get your strings translated, you basically have 2 options.
+
+1st option is to enable [auto-translation in Language settings](http://localhost:3000/blog/tolgee-ai-translator).
+When auto-translation is enabled, Tolgee will automatically translate all new keys created in the Tolgee platform.
+
+2ns option is to use the batch operations feature. You can select all your untranslated keys and translate them in one click.
+
+<video controls>
+  <source src="/img/blog/2023-08-21-autonomous-localization/batch-video.webm" type="video/webm" />
+  Your browser does not support the video tag.
+</video>
+
+## The benefits of autonomous localization
+The most important benefit is definitely cost reduction. Working with translators might become very expensive, and in many cases, the quality is not even higher because many language service providers don't work with context either, and you have to rely on their knowledge of the scope, which is often insufficient.
+
+Another significant benefit is the speed. Working with translators is a complicated process with too many steps. Without
+humans, you can get your translations done in seconds.
+
+## Do I need human translators at all?
+Well, most projects can replace the first translation with machine translation with a Tolgee translator when the context is correctly provided. However, we still recommend some QA checks against the localized app by native speakers to check everything fits. Sometimes the tone of voice or some other details can be slightly off. It's always a good idea to check whether translations haven't broken the layout of the app.
+
+However, we see the future of localization in autonomous localization. Maybe next year, maybe later, the autonomous
+translation would be sufficient to translate most of the apps on the market.
+
+## What is the recommended way to localize my app with Tolgee Autonomous Localization?
+To get maximum from Tolgee Autonomous Localization, you have to use [Tolgee SDKs](http://localhost:3000/js-sdk). Otherwise, the context is not extracted
+from your app. The SDKs are available for all major framework like **React**, **Angular** **Vue.js**, **Svelte** or **plain JavaScript**.
+
+The recommended way is to make developers add the keys to the Tolgee Platform via the [in-context dialog](http://localhost:3000/js-sdk/in-context).
+So they add the key in the source code and add the translation to the Tolgee Platform by `Alt + Clicking` in the app.
+That way, the context is provided, and the translation can be done with the so important context.
+
+You can also add the context "manually" by hitting the update button in the [in-context editor](http://localhost:3000/js-sdk/in-context) of any key in your app. However,
+we know that is not optimal, so we plan to add some "Context gathering mode," so the context will be gathered automatically.
```

Comment:
Do we need to apologize here? I'd present it as an planned feature: `Because Tolgee knows where the translations are located, we plan to add an auto context gathering mode, which will update the context automatically.`